src/react-apps/applications/service-development/src/features/administration/components/Administration.tsx

```diff
@@ -0,0 +1,272 @@
+              <Typography className={classes.headerStyle}>
+                {getLanguageFromKey('administration.administration', this.props.language)}
+              </Typography>
+            </Grid>
+            <Grid item={true} className={classes.mainStyle} md={8}>
+              <AltinnInputField
+                id={'service-name'}
+                onChangeFunction={this.onServiceNameChanged}
+                inputHeader={getLanguageFromKey('general.service_name', this.props.language)}
+                // tslint:disable-next-line:max-line-length
+                inputDescription={getLanguageFromKey('administration.service_name_administration_description', this.props.language)}
+                inputValue={this.state.serviceName}
+                onBlurFunction={this.onBlurServiceName}
+                btnText={getLanguageFromKey('general.edit', this.props.language)}
+                onBtnClickFunction={this.handleEditServiceName}
+              />
+              <AltinnPopper
+                anchorEl={this.state.serviceNameAnchorEl}
+                message={getLanguageFromKey('administration.service_name_empty_message', this.props.language)}
+              />
+              <AltinnInputField
+                id={'repo-name'}
+                inputHeader={getLanguageFromKey('general.service_saved_name', this.props.language)}
+                // tslint:disable-next-line:max-line-length
+                inputDescription={getLanguageFromKey('administration.service_saved_name_administration_description', this.props.language)}
+                inputValue={this.props.service ? this.props.service.name : ''}
+                isDisabled={true}
+              />
+              <AltinnInputField
+                id={'description'}
+                onChangeFunction={this.onServiceDescriptionChanged}
+                inputHeader={getLanguageFromKey('general.service_description_header', this.props.language)}
+                inputDescription={getLanguageFromKey('administration.description_description', this.props.language)}
+                textAreaRows={7}
+                inputValue={this.state.serviceDescription}
+                onBlurFunction={this.onBlurServiceDescription}
+              />
+            </Grid>
+            <Grid item={true} md={4} className={classNames(classes.sidebar)}>
+              <Typography className={classes.sidebarHeader}>
+                {getLanguageFromKey('general.service_owner', this.props.language)}
+              </Typography>
+              <Typography className={classes.sidebarInfoText}>
+                {getLanguageFromKey('administration.service_owner_is', this.props.language)}
+              </Typography>
+              <Typography className={classNames(classes.sidebarServiceOwner, classes.sidebarInfoText)}>
+                <i
+                  className={classNames(classes.iconStyling,
+                    { ['ai ai-corp']: this.props.service.owner.UserType === 2 },
```

Comment:
I thinkn you can put both conditions in the same object (don't need four curly brackets here).

Reply:
removed some crullys, fixed